alf/environments/suite_socialbot.py

```diff
@@ -38,11 +40,61 @@ def is_available():
     return social_bot is not None
 
 
+@alf.configurable
+def transform_reward(reward, reward_cap=1., positive_reward=True):
```

Comment:
Should be a class member of `SparseReward` below, as it is not used by other files / code.